test/unit/rails_sqli_test.rb

```diff
@@ -7,13 +7,13 @@ def test_action_controller_params_are_escaped
     # This should fix # https://groups.google.com/forum/#!topic/rubyonrails-security/8CVoclw-Xkk
     # No error should be raised on that and the SQL
 
-    CMS::Section.find_by(params)
-    sanitized = ActiveRecord::Base.sanitize params['system_name'].to_s
-    assert_match sanitized, CMS::Section.where(params).to_sql
+    CMS::Section.find_by_system_name(params['system_name'].to_s)
```

Comment:
`find_by_system_name` should not be used, those dynamic finders are deprecated!
Also you changed the meaning of the test.

Before we test that when passed a Hash in `where` we sanitize the passed parameters.
Now you already have them passed as a String ... which is not really what the CVE is about